Services/Interfaces/Models/RootLastFm.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Interfaces.Models
+{
+    public class RootLastFm
```

Comment:
What does it mean? Can you name that class properly? names of the class should be understandable

Reply:
f.e. LastFmTopArtistsResponse